cli/main.c

```diff
@@ -334,17 +334,17 @@ char * link_rate_conversion(uint8_t link_rate);
 
 char* major_state_string[11] =
 {
-        "Detect ",
```

Comment:
Changes like this should be squashed, it's not good to add something in one commit, and correct it in another commit in the same PR.